src/accelerate/optimizer.py

```diff
@@ -102,14 +102,14 @@ def zero_grad(self, set_to_none=None):
                 set_to_none = False
             self.optimizer.zero_grad(set_to_none=set_to_none)
 
-    def step(self):
+    def step(self, closure=None):
         if self.state.distributed_type == DistributedType.TPU:
-            xm.optimizer_step(self.optimizer)
+            xm.optimizer_step(self.optimizer, dict(closure=closure))
```

Comment:
```suggestion
            xm.optimizer_step(self.optimizer, optimizer_args={"closure": closure})
```
The [PyTorch XLA doc](https://pytorch.org/xla/release/1.8/index.html#torch_xla.core.xla_model.optimizer_step) seems to suggest we have to use `optimizer_args` to pass this along and it's not the second argument.

Reply:
good catch!